core/src/main/java/edu/softserve/zoo/util/ListUtils.java

```diff
@@ -0,0 +1,27 @@
+package edu.softserve.zoo.util;
+
+import edu.softserve.zoo.exceptions.ApplicationException;
+
+import java.util.List;
+
+/**
+ * <p>This utility class helps to work with lists</p>
+ *
+ * @author Taras Zubrei
+ */
+public class ListUtils {
```

Comment:
You cannot know exactly.
Because it is placed in 'edu.softserve.zoo.util'
It is considered that it is gonna be used in whole app.
==> list.stream().findFirst().orElse(orElse);
No difference at all.
I think it should be more generalized.
You don't need random access in this case, because you're calling stream().